mobile/src/reducers/auth/auth.epics.ts

```diff
@@ -54,16 +65,20 @@ export const listenerAuthStateEpic$ = (action$: ActionsObservable<any>, _: State
     dep.oauthProcess.authenticationState
         .pipe(
             handleHttpErrors(),
-            distinctUntilChanged<AuthenticationState>((x, y) => x.isAuthenticated === y.isAuthenticated),
+            distinctUntilChanged<AuthenticationState>((x, y) => ((x.isAuthenticated === y.isAuthenticated) && x.isAuthenticated)),
```

Comment:
Why is this required? Is x == next, y == previous (consider renaming)?

Reply:
This is needed because if we can't authenticate several times (Authentication state is NotAuthenticated) , we anyway need to show error every time. Without this additional condition no error message will be shown.